package-lock.json

```diff
@@ -8,20 +8,19 @@
       "name": "list",
       "version": "0.1.0",
       "dependencies": {
-        "@testing-library/jest-dom": "^5.16.5",
-        "@testing-library/react": "^13.4.0",
-        "@testing-library/user-event": "^13.5.0",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
         "react-scripts": "5.0.1",
         "web-vitals": "^2.1.4"
+      },
+      "devDependencies": {
+        "eslint": "^8.28.0",
+        "eslint-plugin-react": "^7.31.11",
+        "node-sass": "^7.0.3",
+        "prettier": "2.8.0",
+        "prettier-plugin-apex": "1.11.0"
```

Comment:
why did you add this plugin ?

Reply:
it installed while I was trying to install prettier , but  I uninstalled it now .